scripts/extract_business_rules.py

```
@@ -0,0 +1,201 @@
+                    if func.attr in ('ValidationException', 'BusinessLogicException', 'AlreadyExistsException'):
+                        is_validation = True
+                
+                if is_validation:
+                    msg = None
+                    for keyword in node.exc.keywords:
+                        if keyword.arg == 'msg':
+                            msg = extract_message(keyword.value)
+                    
+                    if not msg and node.exc.args:
+                        if isinstance(node.exc.args[0], ast.Constant) and isinstance(node.exc.args[0].value, str):
+                            if " " in node.exc.args[0].value: 
```

Pylint (4.0.5)

[convention] Trailing whitespace

(C0303)